### PR TITLE
Removing warning caused by radio group

### PR DESCRIPTION
**Pull request by danielstro**
In one the radio group tests, we use an uncontrolled native input and call the click method. This causes the checked property to change in the browser's "state" and the component to mutate without using set state, and so a warning is thrown.
I created a small controlled input component to be used in this test.

### CODE REVIEW

File: test/fixtures/input-fixture.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+export interface InputFixtureProps {
+    value: string;
+    onChange: (e: string) => void;
+}
+
+export interface InputFixtureState {
+    checked: boolean;
+}
+
+export class InputFixture extends React.Component<InputFixtureProps, InputFixtureState> {
```
Comment (JoMarton) [inline test/fixtures/input-fixture.tsx]: Please rename class, props, state and file name to contain the word "radio" since this is a radio button input

**Comment by tobich** [comment]
It is not clear to me neither from the description nor from the code, what the problem was and how it the code change fix it.

**Comment by danielstro** [comment]
I changed the description, I hope it is better now.

**Comment by JoMarton** [comment]
Refactoring RadioGroup on a different branch and this fix is included in it

**Pull request closed without merging**